### PR TITLE
Bump to typescript@3.7 and using eslint instead of tslint

### PR DESCRIPTION
**Pull request by nodece**

### CODE REVIEW

**Comment by nodece** [comment]
Should be error in here. Because typescript 3.7 is not supported by prettier.

File: package.json [hunk 1]
```diff
@@ -1,31 +1,36 @@
 {
   "name": "casbin",
-  "version": "3.0.5",
+  "v  ersion": "3.0.5",
```
Comment (dreamdevil00) [inline package.json]: ? version

Comment (hsluoyz) [inline package.json]: Fix this.

Comment (nodece) [reply package.json]: yes

**Pull request merged**